apps/air-discount-scheme/web/screens/Admin/utils.ts

```diff
@@ -7,9 +7,7 @@ import { FilterInput, financialStateOptions } from './consts'
 
 const getAirlinesAvailableForCSVDownload = (filters: FilterInput) =>
```

Comment:
ditto on this file.... since e.g. the first month will differ, could be some flights that still have a cooperation.